bin/jam/args.ts

```diff
@@ -29,6 +29,9 @@ Options:
                         [default: ${NODE_DEFAULTS.config}]
   --pvm                 PVM Backend, one of: [${PvmBackendNames.join(", ")}].
                         [default: ${PvmBackendNames[NODE_DEFAULTS.pvm]}]
+  --accumulate-sequentially
+                        Disable parallel accumulation.
+                        [default: false]
```

Comment:
I'd be in favor of not exposing that option via CLI of the main node. At most it should be in the config json, since it's a debugging parameter, but imho even this should be a separate task. I believe we currently need that only for the test runner, right?

Reply:
I removed it from all places expect test-runner